com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/pipeline/LuaBuilder.java

```diff
@@ -288,29 +288,36 @@ public byte[] constructLuaBytecode(Task<Void> task, String luacExe, String sourc
     }
     */
 
-    public byte[] constructLuaJITBytecode(Task<Void> task, String luajitExe, String source) throws IOException, CompileExceptionError {
+    public byte[] constructLuaJITBytecode(Task<Void> task, String luajitExe, String source, boolean gen32bit) throws IOException, CompileExceptionError {
 
         Bob.initLua(); // unpack the lua resources
 
         File outputFile = File.createTempFile("script", ".raw");
         File inputFile = File.createTempFile("script", ".lua");
 
-        // Doing a bit of custom set up here as the path is required.
-        //
-        // NOTE: The -F option for bytecode is a small custom modification to bcsave.lua in LuaJIT which allows us to supply the
-        //       correct chunk name (the original source file) already here.
-        //
-        // See implementation of luaO_chunkid and why a prefix '@' is used; it is to show the last 60 characters of the name.
-        //
-        // If a script error occurs in runtime we want Lua to report the end of the filepath
-        // associated with the chunk, since this is where the filename is visible.
+        // -b = generate bytecode
+        // 
+        // -W = Generate 32 bit (non-GC64) bytecode.
+        // -X = Generate 64 bit (GC64) bytecode.
+        // -d = Generate bytecode in deterministic manner.
```

Comment:
Converting the same .lua file to bytecode multiple times will always return exactly the same bytecode. This is very useful when working with LiveUpdate.